Source/MSBuild.Sdk.Extras/Build/MSBuild.NET.Inference.targets

```diff
@@ -114,4 +106,10 @@
     <Import Condition="'$(TargetFrameworkVersion)' != ''" Project="$(MSBuildThisFileDirectory)FrameworkResolvers\MSBuild.NETPortable.v$(TargetFrameworkVersion).targets"/>
   </ImportGroup>
 
+  <PropertyGroup>
+    <_TargetFrameworkVersionWithoutV>$(TargetFrameworkVersion)</_TargetFrameworkVersionWithoutV>
+    <_TargetFrameworkVersionWithV>v$(TargetFrameworkVersion)</_TargetFrameworkVersionWithV>
+    <TargetFrameworkVersion>$(_TargetFrameworkVersionWithV)</TargetFrameworkVersion>
+  </PropertyGroup>
```

Comment:
> Can it be `
<Import Condition="'$(TargetFrameworkVersion)' != ''" Project="$(MSBuildThisFileDirectory)FrameworkResolvers\MSBuild.NETPortable.$(TargetFrameworkVersion).targets"/>`

We can do that.

> Reassigning the variable like this may lead to subtle errors and make following the values in the trace harder.

But here this is the single point of change where the prefixing of `v` actually happens, everywhere else it's just version, so tracing is easy. Also when the import reaches platform targets, we can be confident that any assigned `TFV` will be valid `vX.X` format!